src/trace/xray-service.ts

```diff
@@ -172,11 +181,11 @@ export class XrayService {
 
   private convertToParentId(xrayParentId: string): string | undefined {
     if (xrayParentId.length !== 16) return;
-
-    const hex = new BigNumber(xrayParentId, 16);
-    if (hex.isNaN()) return;
-
-    return hex.toString(10);
+    try {
+      return BigInt("0x" + xrayParentId).toString(10);
+    } catch (_) {
+      return undefined;
```

Comment:
Maybe we can add a debug statement here?

Reply:
will do. i'll add `logDebug("Failed to parse xrayParentId")`